### DISCUSSION TITLE
Splitwise Firefly Sync

### DISCUSSION THREAD

**Discussion by adyanth** [firefly-iii/firefly-iii#6796] [General]
I use Splitwise quite frequently with my roommates, friends and family. Entering a transaction on Splitwise and Firefly III can get taxing at times, so here is a tool that can sync transactions from Splitwise to Firefly using their respective APIs. Designed to be run as a cron job every day, there are still a couple of [issues](https://github.com/adyanth/splitwise-firefly-sync/issues) preventing me from running it as a cron job.

For now, it works well for me by validating a dry run first and then running the sync for a week.

Tool: https://github.com/adyanth/splitwise-firefly-sync

**Comment by JC5** [comment]
That is very cool! Can I add it to the documentation?

**Reply by adyanth** [reply]
Sure! It can now be run as a cron job without getting duplicate entries, and once the 5.8.0 is out, it can prevent dummy updates if the entries are already synced too :)

**Reply by Jonty16117** [reply]
Wow thank you for this....this is one of the major selling point of firefly
